src/libraries/System.Private.CoreLib/src/System/Security/SecureString.cs

```diff
@@ -429,7 +429,9 @@ private sealed class UnmanagedBuffer : SafeBuffer
             // A local copy of byte length to be able to access it in ReleaseHandle without the risk of throwing exceptions
             private int _byteLength;
 
+#pragma warning disable CA1419
```

Comment:
Would it be better to add the extra constructor, and leave it for the IL linker to delete if it is unused?

Otherwise, it would be nice to add a comment to make it easy to see what this is disabling.

Reply:
IMO we should just make the private parameterless constructor here public as per the analyzer/code fix. Either that or put a comment about why we’re not doing that.

Reply:
I'll add a comment.  This type is not for use with P/Invoke, and no one should be using a parameterless ctor other than the type itself.  Adding a ctor would be very misleading.